go2_robot_sdk/go2_robot_sdk/go2_driver_node.py

```diff
@@ -89,7 +89,7 @@ def __init__(self):
         self.publish_raw_voxel = self.get_parameter(
             'publish_raw_voxel').get_parameter_value().bool_value
 
-        self.conn_mode = "single" if len(self.robot_ip_lst) == 1 else "multi"
+        self.conn_mode = "single" if (len(self.robot_ip_lst) == 1 and self.conn_type != "cyclonedds") else "multi"
```

Comment:
cyclonedds defines their stuff as multi even in case of a single robot

see: https://github.com/abizovnuralem/go2_ros2_sdk/blob/cd73e833e0372d7eb2174c8bafaeb12a9aa0d0c5/go2_robot_sdk/go2_robot_sdk/go2_driver_node.py#L281